workers/common/src/test/kotlin/common/JsonSchemaTest.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.workers.common.common
+
+import com.networknt.schema.JsonSchemaFactory
+import com.networknt.schema.SpecVersion
+import com.networknt.schema.serialization.JsonNodeReader
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.inspectors.forAll
+import io.kotest.matchers.collections.beEmpty
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldNot
+
+import java.io.File
+
+import org.ossreviewtoolkit.model.yamlMapper
+
+class JsonSchemaTest : WordSpec({
+    val validConfigs = File("src/test/resources")
+        .listFiles()
```

Comment:
For portability, prefer Kotlin's `walk().maxDepth(1)` over Java's `listFiles()`. Same below.

Reply:
That also allows to drop the `toList()` calls.